Lib/logging/__init__.py

```diff
@@ -1880,7 +1880,9 @@ class LoggerAdapter(object):
     information in logging output.
     """
 
-    def __init__(self, logger, extra=None):
+    merge_extras = False
+
+    def __init__(self, logger, extra=None, merge_extras=None):
```

Comment:
I'd use `merge_extra=False` here - `merge_extra` to align with `extra`, and `False` as a default value as there's no need to consider the `None` case.